.github/workflows/pytest-cpu.yaml

```diff
@@ -89,7 +89,7 @@ jobs:
           name: coverage-${{ github.sha }}-${{ inputs.name }}
           path: .coverage
       - name: Notify slack fail
-        if: failure() && !cancelled() && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/dev') && (github.event_name != 'pull_request' || github.event_name != 'pull_request_target')
+        if: failure() && !cancelled() && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/dev') && (github.event_name != 'pull_request' && github.event_name != 'pull_request_target')
```

Comment:
was this intentional? i don't think the event name can be both pull_request and pull_request_target

Reply:
This was buggy before -- it should only run if its not pull request and not pull request target (otherwise, it always evaluates true)